### PR TITLE
Icons: Fix typescript setup

### PR DESCRIPTION
**Pull request by peterhass**
We need all our exports as commonjs. Previously we had esmodules in there

### CODE REVIEW

**Review by eliasbraun — APPROVED** [review]
Looks good, but maybe wait for another approval! 
I think you forgot the Version Bump `4.0.1` or was it intended?

File: packages/icons/package.json
Comment (peterhass) [reply packages/icons/package.json:19]: Because the script is doing it now

**Pull request merged**
